### DISCUSSION TITLE
Numeric keypad not working when InputOtp has [integerOnly]="true"

### DISCUSSION THREAD

**Discussion by sannila** [primefaces/community#1593] [PrimeNG] [answered]
In PrimeNG, **InputOtp**
When we providing [integerOnly]="true", numeric pad (right side of the keybord) is not working.
Note: Num Lock is Off

![image](https://github.com/primefaces/community/assets/41979819/e23e1ac1-85d7-4fdc-919e-13b8faa1af83)

Do we have any option to fix this.

**Comment by mehmetcetin01140** [comment, marked as answer]
Hi,

Issue link: https://github.com/primefaces/primeng/issues/15195
This issue has been fixed. Could you please try again with PrimeNG version 17.13.0?”
